build.gradle.kts

```diff
@@ -0,0 +1,95 @@
+plugins {
+    java
+    id("org.embulk.embulk-plugins") version("0.6.2")
+    id("com.diffplug.spotless") version( "6.25.0")
+
+}
+
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(17)
```

Comment:
This seems a bit unsafe with the current version of embulk. Well, most cases should work with Java 17.
if you run embulk with Java 17, the plugin should work, too.